examples/cpp/desktop-testbed/2_run_desktop.sh

```
@@ -0,0 +1 @@
+./build/desktop-testbed-ubuntu-x64-debug/desktop-testbed
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)